frontend/src/result-list.js

```diff
@@ -72,7 +72,7 @@ const ResultList = () => {
   const [runInputValue, setRunInputValue] = useState('');
   const [runFilterValue, setRunFilterValue] = useState('');
 
-  const [resultSelection, setResultSelection] = useState([]);
+  const [resultSelection, setResultSelection] = useState('');
```

Comment:
I'm not confident this is the right choice here, since multiple functions in the module treat this as a list.

Consider instead changing the single line that expects a string to check for an empty list:
```
      {resultSelection.length !== 0 ? resultSelection : 'Select a result'}
```